modules/react/modal/stories/stories.tsx

```diff
@@ -39,50 +39,58 @@ export const Default = () => {
 
 export const WithTooltips = () => {
   const {targetProps, modalProps, closeModal} = useModal();
-  const popup1 = usePopup();
-  const popup2 = usePopup();
+  const popup1 = usePopupModel();
+  const popup2 = usePopupModel();
 
-  useCloseOnOutsideClick(popup1.stackRef, popup1.closePopup);
+  useCloseOnOutsideClick(popup1);
 
   return (
     <>
       <DeleteButton {...targetProps}>Delete Item</DeleteButton>
-      <Modal data-testid="TestModal" heading={'Delete Item'} {...modalProps}>
+      <Modal heading={'Delete Item'} {...modalProps}>
         <p>Are you sure you'd like to delete the item titled 'My Item'?</p>
-        <SecondaryButton {...popup1.targetProps}>Open Popup to reach Delete button</SecondaryButton>
-        <SecondaryButton {...popup2.targetProps}>Non-hidable Popup</SecondaryButton>
+        <Popup.Target model={popup1}>Open Popup to reach Delete button</Popup.Target>
```

Comment:
Nit: Can we add some spacing to this story in both popups to make things look a little nicer?

Reply:
Hmm. This is a testing story